inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Epinow2",
+    "definitions": {
+        "Epinow2": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "job_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "task_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "as_of_date": {
+                    "type": "string",
+                    "format": "date"
+                },
+                "disease": {
+                    "type": "string"
+                },
+                "geo_value": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "geo_type": {
+                    "type": "string"
+                },
+                "parameters": {
+                    "$ref": "#/definitions/Parameters"
+                },
+                "data": {
+                    "$ref": "#/definitions/Data"
+                },
+                "seed": {
+                    "type": "integer"
+                },
+                "horizon": {
+                    "type": "integer"
+                },
+                "priors": {
+                    "$ref": "#/definitions/Priors"
+                },
+                "sampler_opts": {
+                    "$ref": "#/definitions/SamplerOpts"
+                }
+            },
+            "required": [
+                "as_of_date",
+                "data",
+                "disease",
+                "geo_type",
+                "geo_value",
+                "horizon",
+                "job_id",
+                "parameters",
+                "priors",
+                "sampler_opts",
+                "seed",
+                "task_id"
+            ],
+            "title": "Epinow2"
+        },
+        "Data": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["null", "string"]
+                },
+                "report_date": {
```

Comment:
No, I was thinking of this as the task config -- only corresponds to one report date. But I wanted to keep this specification easy-ish to extend to other models and Epinowcast takes multiple report dates. So I put report date as a list of one report date rather than just one date.

Do you think that's overengineering here? I could just force it to be one report date and we can change the spec down the road if needed.

Reply:
I think it's fine as is